libs/pilight/core/json.c

```diff
@@ -1436,3 +1436,28 @@ int json_find_string(JsonNode *object, const char *name, char **out) {
 void json_free(void *a) {
 	free(a);
 }
+
+JsonNode *json_vartype(int vartype) {
```

Comment:
I think it would be better to change this to a function that can be used like this:
`json_append_element(vartype_array, json_mkstring(json_vartype(JSON_ARRAY)))`